kanidm_unix_int/src/daemon.rs

```diff
@@ -439,7 +440,7 @@ async fn main() {
     });
 
     // Check the pb path will be okay.
-    if cfg.db_path != "" {
+    if cfg.db_path.is_empty() {
```

Comment:
At this point, you're changing this from "if cfg.db_path is set" to "if cfg.db_path is unset" - this inverts the test.

Reply:
At the least make it `!cfg.db_path.is_empty()`